src/locale/languages/es.js

```diff
@@ -0,0 +1,322 @@
+// Hash: 01ae4880b9b4296907d680e9b909713c36c42491
+// Link: https://github.com/gitpoint/git-point/pull/260
+
+export const erase = {
+  auth: {
+    login: {
+      welcomeTitle: 'Bienvenido a GitPoint',
+      welcomeMessage:
+        'El cliente GitHub más rico en funciones que es 100% gratuito',
+      notificationsTitle: 'Control notifications',
+      notificationsMessage:
+        'View and control all of your unread and participating notifications',
+      reposTitle: 'Repositorios y Usuarios',
+      reposMessage:
+        'Easily obtain repository, user and organization information',
+      issuesTitle: 'Issues y Pull Requests',
+      issuesMessage:
+        'Communicate on conversations, merge pull requests and more',
+      signInButton: 'SIGN IN',
+    },
+    welcome: {
+      welcomeTitle: 'Bienvenido a GitPoint',
+    },
+    events: {
+      welcomeMessage:
+        "Welcome! This is your news feed - it'll help you keep up with recent activity on repositories you watch and people you follow.",
+      commitCommentEvent: 'commented on commit',
+      createEvent: 'creado {{object}}',
+      deleteEvent: 'borrado {{object}}',
```

Comment:
Shouldn't these two lines be `{{object}} creado` and `{{object}} borrado`?

Reply:
Makes perfect sense! IDK what I was thinking lol